main/lsp/LSPTypechecker.h

```diff
@@ -71,6 +72,8 @@ class LSPTypechecker final {
     /** Used for assertions. Indicates if `initialize` has been run. */
     bool initialized = false;
 
```

Comment:
You should remove the old `diagnosticEpochs` and `filesWithErrors` fields from this class now that they are in `ErrorReporter`.

Reply:
I've left them in for now until we figure out what to do about `UndoState` so that I don't get errors in `LSPTypechecker.cc`